js/admin.js

```diff
@@ -29,7 +29,7 @@
 		$.ajax({
 			type: 'POST',
 			dataType: 'json',
-			url: ajaxurl,
+			url: window.adstxt.ajaxUrl,
```

Comment:
I don't think this is what was intended by the original ticket, as the admin-ajax URL is currently working as expected.